Applications/SseServer/test/SseServerTests/EventQueueTests.cs

```diff
@@ -97,6 +99,27 @@ public async Task Dequeue_only_returns_elements_for_the_given_identity()
         eventNames.ShouldHaveCount(1);
         eventNames.ShouldContain("event1");
     }
+
+    [Fact]
+    public async Task Queue_sends_keep_alive_event()
+    {
+        // Arrange
+        var config = new Configuration { Sse = new Configuration.SseConfiguration { KeepAliveEventInterval = 1 } };
+        var options = A.Fake<IOptions<Configuration>>();
+        A.CallTo(() => options.Value).Returns(config);
+
+        var queue = new EventQueue(A.Fake<ILogger<EventQueue>>(), options);
+        queue.Register("testAddress", CancellationToken.None);
+
+        // Act
+        await Task.Delay(TimeSpan.FromMilliseconds(500), TestContext.Current.CancellationToken);
+        var eventNames = await queue.DequeueAllFor("testAddress");
+        queue.Deregister("testAddress");
```

Comment:
Typically the act part has only one statement. I see that it is hard here. But I'd say that the one statement is the `Task.Delay`. Everything that comes after should go to Assert. 
Actually, the Deregister shouldn't even be necessary. The queue is disposed after the test anyway.